prebid-video/video-long-form.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: page_v2
+title: Getting Started with Long-Form Video for Prebid.js
+description: Prebid Long-Form Video Getting Started
+pid: 1
+is_top_nav: yeah
+top_nav_section: pbjs-video
+nav_section: pbjs-video-get-started
+sidebarType: 4
+---
+
+# Getting Started with Long-Form Video
+{: .no_toc }
+
+If you’re new to header bidding and Prebid review the following to get a general understanding of Prebid.js
+
+-	[What Is Prebid?]({{site.github.url}}/overview/intro.html)
+-	[Getting Start with Prebid]({{site.github.url}}/overview/getting-started.html)
+
+See [Prebid.js Video Overview]({{site.github.url}}/prebid-video/video-overview.html) for a general description and high-level overview of working with video demand in Prebid.js.
+
+* TOC
+{:toc}
+
+## Ad Ops
+
+### Ad Server Setup
+
+Start by reading [AdOps Getting Started]({{site.github.url}}/overview/getting-started.html). This will give you a general overview of setting up your price buckets and line items on your ad server.
+
+One thing to keep in mind as you set up your line items is price granularity. Be sure to communicate your price granularity requirements to your developers, as they might need to define custom configuration settings, depending on your granularity.
+
+<span style="color:#ff0000">ANYTHING SPECIAL REQUIRED FOR LONG-FORM Video?</span>
```

Comment:
yes we will have to share freewheel setup steps. I will discuss and let you know soon